ansibotmini.py

```diff
@@ -1157,9 +1161,21 @@ def match_version(obj: GH_OBJ, actions: Actions, ctx: TriageContext) -> None:
         return
     if match := VERSION_RE.search(obj.body):
         if match := VERSION_OUTPUT_RE.search(match.group(1)):
-            actions.to_label.append(
-                f"affects_{'.'.join(match.group(1).split('.')[:2])}"
-            )
+            major_version = ".".join(match.group(1).split(".")[:2])
+            actions.to_label.append(f"affects_{major_version}")
+            if (
+                "bug" in actions.to_label
+                and major_version not in ctx.supported_bugfix_versions
+            ):
```

Comment:
We should probably do this only for new issues to prevent spamming half the repo.